src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Raven.Client.Exceptions.Commercial;
+using Raven.Client.ServerWide;
+using Raven.Server.Commercial;
+using Raven.Server.Json;
+using Raven.Server.ServerWide.Commands;
+using Raven.Server.ServerWide.Commands.ETL;
+using Raven.Server.ServerWide.Commands.Indexes;
+using Raven.Server.ServerWide.Commands.PeriodicBackup;
+using Raven.Server.ServerWide.Context;
+using Voron.Data.Tables;
+
+namespace Raven.Server.ServerWide;
+
+public sealed partial class ClusterStateMachine
+{
+    private const int MinBuildVersion54111 = 54_111;
+
+    private void AssertLicenseLimits(string type, ServerStore serverStore, DatabaseRecord databaseRecord, Table items, ClusterOperationContext context)
+    {
+        switch (type)
+        {
+            case nameof(PutIndexCommand):
+            case nameof(PutIndexesCommand):
+                if (CanAssertLicenseLimits(context, minBuildVersion: MinBuildVersion54111) == false)
```

Comment:
You can remove `CanAssertLicenseLimits`. We are going to check it for all versions.

Reply:
remove completely or just for indexes?

Reply:
Let's leave it and update the version (to the latest build number).

Reply:
Done